code/src/components/AddTodo.js

```diff
@@ -0,0 +1,67 @@
+import React, { useState } from "react";
+import { useDispatch } from "react-redux";
+import styled from "styled-components";
+
+import todos from "../reducers/todos";
+
+const TextForm = styled.form`
+  margin-right: 5px;
+  input {
+    width: 170px;
+    height: 30px;
+    border-bottom: 5px solid pink;
+    border-radius: 5px;
+    text-align: center;
+    border: none;
+    font-size: 20px;
+    font-family: "Caveat", cursive;
+    @media (min-width: 768px) {
+      width: 410px;
+      margin: 50px auto;
+      font-size: 30px;
+    }
+  }
+`;
+const AddButton = styled.button`
+  width: 40px;
+  height: 40px;
+  color: pink;
+  font-size: 25px;
+  background-color: white;
+  border: none;
+  border-radius: 50%;
+  margin: 5px;
+  cursor: pointer;
```

Comment:
NIce structure in the style-component! Cursor pointer I don´t know what it does:)